apps/frontend/src/shared/components/ui/Tooltip.tsx

```
@@ -0,0 +1,76 @@
+
+  const arrowClasses = {
+    top: 'top-full left-1/2 -translate-x-1/2 -mt-1.5 border-t-gray-900 border-r-transparent border-b-transparent border-l-transparent border-t-[6px] border-r-[6px] border-b-0 border-l-[6px]',
+    bottom:
+      'bottom-full left-1/2 -translate-x-1/2 -mb-1.5 border-b-gray-900 border-r-transparent border-t-transparent border-l-transparent border-b-[6px] border-r-[6px] border-t-0 border-l-[6px]',
+    left: 'left-full top-1/2 -translate-y-1/2 -ml-1.5 border-l-gray-900 border-t-transparent border-b-transparent border-r-transparent border-l-[6px] border-t-[6px] border-b-[6px] border-r-0',
+    right:
+      'right-full top-1/2 -translate-y-1/2 -mr-1.5 border-r-gray-900 border-t-transparent border-b-transparent border-l-transparent border-r-[6px] border-t-[6px] border-b-[6px] border-l-0',
+  }
+
+  return (
+    <div className={cn('relative inline-block', className)} onMouseEnter={() => setIsHovered(true)} onMouseLeave={() => setIsHovered(false)}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)